### ISSUE TITLE
Inherited method description does not escape HTML

### ISSUE THREAD

**Issue by garysoed** [yui/yuidoc#324]
For example:
```javascript
/**
 * @class ability.Ability
 */

/**
 * Returns the trigger for this ability on this element.
 * @method getTrigger
 */

/**
 * @constructor
 * @class ability.Droppable
 * @extends ability.Ability
 */
```

output in ability.Droppable:
```
<p>Returns the trigger for this ability on this element.</p>
```

**Comment by okuryu** [comment]
Thanks! It seems to be a bug. I'll take a look.

**Comment by hhff** [comment]
hi @okuryu - reporting this as well:

![screen shot 2015-03-29 at 6 05 23 pm](https://cloud.githubusercontent.com/assets/2865404/6888009/346a6c9a-d63e-11e4-87f8-e12f12ea16b9.png)

**Comment by okuryu** [comment]
@hhff Thanks! I'm working on this. :)

**Comment by okuryu** [comment]
Published `yuidocjs@0.7.0`.

**Comment by vmaudgalya** [comment]
Hi @okuryu I'm using `yuidocjs@0.10.0`

I've found that when adding the comments:
```javascript
/**
* Gets user info
*
* @method getUserInfo
* @return {Object} Returns JSON object containing todos and notifications for the user.
*/
```

The generated HTML is as follows:
```html
<div class="returns" id="yui_3_9_1_2_1456452399664_459">
  <h4>Returns:</h4>
    <div class="returns-description" id="yui_3_9_1_2_1456452399664_458">
      <span class="type"><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object" class="crosslink external" target="_blank">Object</a></span>:
         <p>Returns JSON object containing todos and notifications for the user.</p>
    </div>
</div>
```

You can see that there's an extra `<p>` tag, in the "Returns" statement. Is this a bug, or is there some way to suppress this?
